### PR TITLE
#20 angular card styling

### PR DESCRIPTION
**Pull request by prateekgirdhar**

### CODE REVIEW

File: ReStyle-Frontend/src/app/components/header/header.component.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import url('https://fonts.googleapis.com/css?family=Nunito');
+
+.header{
+  font-family: 'Nunito', sans-serif;
+  font-size: 1.5em;
+  letter-spacing: 0.1em;
+  font-style: italic;
+  font-weight: 800;
+  justify-content: center;
+}
```
Comment (ZedTT) [inline ReStyle-Frontend/src/app/components/header/header.component.css]: Should end in newline

File: ReStyle-Frontend/src/app/components/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<nav class="header navbar navbar-dark fixed-top" style="background-color: #da8678;" data-offset-top="300">
+  ReStyle
+</nav>
```
Comment (ZedTT) [inline ReStyle-Frontend/src/app/components/header/header.component.html]: should end in newline

File: ReStyle-Frontend/src/app/components/header/header.component.css (hunk 1 above)
Comment (prateekgirdhar) [reply ReStyle-Frontend/src/app/components/header/header.component.css]: Thanks, fixed now!

File: ReStyle-Frontend/src/app/components/header/header.component.html (hunk 2 above)
Comment (prateekgirdhar) [reply ReStyle-Frontend/src/app/components/header/header.component.html]: Fixed!

File: ReStyle-Frontend/src/app/components/item-card/item-card.component.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ * Generated by Animista on 2019-5-3 14:44:13
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+
+/**
+ * ----------------------------------------
+ * animation slide-out-right
+ * ----------------------------------------
+ */
+ @-webkit-keyframes slide-out-right {
```
Comment (ZedTT) [inline ReStyle-Frontend/src/app/components/item-card/item-card.component.css]: Something about the slide out right animation creates a horizontal scroll bar. This needs to be fixed. The left slide out works fine

Comment (prateekgirdhar) [reply ReStyle-Frontend/src/app/components/item-card/item-card.component.css]: Yes, I see that too. I'll look into it.

**Pull request merged**